migrations/V45__Add_Notes_Table.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE note (
+    id uuid PRIMARY KEY not null,
+    system_intake uuid not null REFERENCES system_intake(id),
+    created_at timestamp with time zone,
+    eua_user_id text not null,
+    author_name text,
```

Comment:
We might also want the email address for this person, BUT we don't need it yet, and we can add it in the future. So, mostly making a note here that I see the potential and am purposefully ignoring it for now.